### ISSUE TITLE
Disable autoload nextcloud credentials in new http download window

### ISSUE THREAD

**Issue by jsalatiel** [e-alfred/ocdownloader#92]
This is more behaviour change request, so  i think the issue template is not required.
Well, when using ocdownloader and my username/password for nextcloud is saved on firefox these credentials will be autocompleted by default in the ocdownloader. Can this be disabled?

![image](https://user-images.githubusercontent.com/5522389/43666113-e3dceaee-9748-11e8-9f35-97cee0d923c7.png)

The way it is , it is very easy send my nextcloud credentials to a remote site by mistake.

**Comment by RT4U** [comment]
Yes, this is a real security issue. Just by accident the nextcloud admin credentials are sent through the remote server. I am hoping for a quick fix too
